### ISSUE TITLE
Code Review - Timothy P. McCrary

### ISSUE THREAD

**Issue by tpmccrary** [Jesusmaxhernandez/Lab1-CS2302#2]
**Good**
- Code looks clean, well indented and spaced.
- Code is commented and comments are clear and understandable.
- Good variable names, purpose is easily recognized.
- Code isn't overly complicated.

**Things to work on**
- Instead of saying you did other test, show it in the code(i.e. create files that have properties like the tests you described, run, and show them).
- Instead of having the commented out code segment that is meant to print the dog and cat list, put that code in its own function and call it ( or don't call it) as you need.

In summary: code looks good, good job!
